### PR TITLE
Diona can now have viruses cured by radiation exposure or radium injections

### LINKED ISSUE #7875: Diona virus issue
Diona can currently be infected by viruses, but swallowing an antibody pill, or grinding one up and injecting them with it, does not cure the virus.

### PR DESCRIPTION
**Pull request by mikomyazaki**
Alternative to #8022 which got shot down for lore reasons.

Fixes #7875 

Radium or mild radiation exposure will cure viruses after several life ticks (expected) in Diona.

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!bugfix

**Comment by mikomyazaki** [comment]
!review

File: html/changelogs/doona_virus_cures.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: ChangeMe
```
Comment (NonQueueingMatt) [inline html/changelogs/doona_virus_cures.yml]: gotta change the author boss

Comment (mikomyazaki) [reply html/changelogs/doona_virus_cures.yml]: oops

**Pull request merged**